terragrunt/aws/central_account/main.tf

```
@@ -8,7 +8,7 @@ locals {
 
 # Role used by Terraform to manage all satellite accounts
 module "gh_oidc_roles" {
-  source = "github.com/cds-snc/terraform-modules?ref=v3.0.19//gh_oidc_role"
+  source = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v6.1.4"
```

Same comment about the version revert:
```suggestion
  source = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v3.0.19"
```

---

This has been updated!